### PR TITLE
Sentiment_Anaylysis_Model

### PR DESCRIPTION
**Pull request by Iamlegend-Imani**
Naming: feature/descriptive-name

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
The TwitterBot needs to be able to understand the different responses from the twitter users about Police Misconduct.

  What specific changes were made?
Jupyter notebook containing an LSTM Model

Submitter Checklist: 
  - [x] Small scope (1-2 components)
  - [x] Not too many concerns in a single commit
  - [x] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [ ] Atomic, descriptive commit messages
  - [ ] Consistent formatting

Reviewer Checklist:
  - [x] Small scope (1-2 components)
  - [x] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [x] Give feedback no matter what conversations happened elsewhere, document here
  - [ ] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

**Comment by rhiag** [comment]
Pair programmed with Imani and Andrew on this. At the moment although the model has a great training accuracy, the testing accuracy is only about 50%. We will continue to try and improve this model.

**Review by Andrew-Haney — COMMENTED** [review]
Looks good! I know that it was mentioned about going to binary answers so we could clean up the code to avoid any further confusion with next cohort(s).

**Comment by rhiag** [comment]
Update: We have changed from multiclass sentiments to binary sentiments and it has improved the model accuracy quite a bit. We want to incorporate more data and explore the use of Distilbert pretrained model to see if this makes a difference as well

**Comment by Iamlegend-Imani** [comment]
Update: Added more data, Model Accuracy did not chance. We are looking to now explore the use of Distilbert pretrained model to see if this makes any difference!

**Review by rhiag — APPROVED** [review]
@Iamlegend-Imani and I Cleaned up the Sentiment Analysis model. The comments in the notebook should give the next team insight into what we worked on.

File: app/scraper.py
Comment (JAaron93) [inline app/scraper.py:80]: If you can't stash your changes to scraper.py, you should think about deleting this branch and making a new one. Considering you're working in a Colab notebook you already have a copy sitting in Drive. So when you do make a new branch, you can add a copy of this notebook to it and avoid deleting the scraper altogether. Easy fix.

File: notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb
Comment (Andrew-Haney) [inline notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb:1539]: Not sure if this was a maybe that was missed or if it was intentionally left in but I'd guess this should be a no instead of a maybe.  Please fix before final PR.

File: notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb
Comment (Andrew-Haney) [inline notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb:1951]: If not using GloVe can we delete line 1951 and maybe just include a note about trying GloVe in the documentation we're leaving for the next cohort?

File: notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb
Comment (Andrew-Haney) [inline notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb:2780]: Again, just being picky here but please leave an empty line after the final line in code.

**Review by Andrew-Haney — COMMENTED** [review]
Loving the results we're getting from this! I noted a few minor issues that should be cleaned up before final PR and merge.  Well done everybody!

**Comment by Iamlegend-Imani** [comment]
Closing out this pull request because of the conflicting file (scraper.py) instead of deleting it so that the model's progress is available to view.
I will be creating a new pull request

**Comment by Iamlegend-Imani** [comment]
Please DO NOT Merge this Pull Request

**Pull request closed without merging**
